PackageExplorer/Converters/DateTimeOffsetConverter.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Globalization;
+using System.Windows.Data;
+
+namespace PackageExplorer
+{
+    public class DateTimeOffsetConverter : IValueConverter
+    {
+        #region IValueConverter Members
+
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            if (value is DateTimeOffset)
+            {
+                return ((DateTimeOffset)value).DateTime.ToLocalTime().ToString();
+            }
+            return null;
+        }
+
+        public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            throw new NotImplementedException();
```

Comment:
A lot of the other converters also throw a `NotImplementedException`. See:
- https://github.com/NuGetPackageExplorer/NuGetPackageExplorer/blob/master/PackageExplorer/Converters/AndLogicConverter.cs#L25
- https://github.com/NuGetPackageExplorer/NuGetPackageExplorer/blob/master/PackageExplorer/Converters/BooleanToStringConverter.cs#L19
- https://github.com/NuGetPackageExplorer/NuGetPackageExplorer/blob/master/PackageExplorer/Converters/BooleanToVisibilityConverter.cs#L41

...

But if you want I can change it.